Error when using inside repl.it

## Details

If you try to require module inside [repl.it](https://repl.it), following error occurs:

### JS input

```js
const polyfillLibrary = require('polyfill-library');
```

### Console output

```sh
node v9.7.1 linux/amd64
   

Repl.it: Installing fresh packages
Repl.it: 


> polyfill-library@3.27.4 postinstall /home/runner/node_modules/polyfill-library
> npm run build


> polyfill-library@3.27.4 build /home/runner/node_modules/polyfill-library
> npm run clean && node tasks/node/updatesources && node tasks/node/buildsources


> polyfill-library@3.27.4 clean /home/runner/node_modules/polyfill-library
> rimraf ./polyfills/__dist

Updating third-party polyfills...
_TypedArray: New polyfill
  from ../js-polyfills/typedarray.js
~html5-elements: New polyfill
  from ../html5shiv/dist/html5shiv.min.js
atob: New polyfill
  from ../Base64/base64.js
AudioContext: New polyfill
  from ../audio-context-polyfill/AudioContext.js
EventSource: New polyfill
  from ../event-source-polyfill/eventsource.js
fetch: New polyfill
  from ../whatwg-fetch/fetch.js
 * Running module-specific update task update.task.js
HTMLPictureElement: New polyfill
  from ../picturefill/dist/picturefill.min.js
  from ../picturefill/dist/plugins/mutation/pf.mutation.min.js
Intl: New polyfill
  from ../intl/dist/Intl.min.js
 * Running module-specific update task update.task.js
Importing Intl.~locale.* polyfill from /home/runner/node_modules/intl/locale-data/jsonp
716 imported locales
Intl polyfill imported successfully
JSON: New polyfill
  from ../json3/lib/json3.js
MutationObserver: New polyfill
  from ../mutationobserver-shim/MutationObserver.js
Promise: New polyfill
  from ../yaku/dist/yaku.browser.global.min.js
UserTiming: New polyfill
  from ../usertiming/src/usertiming.js
WebAnimations: New polyfill
  from ../web-animations-js/web-animations.min.js
Polyfills updated successfully
Writing compiled polyfill sources to /home/runner/node_modules/polyfill-library/polyfills/__dist/...
{"errno":-24,"code":"EMFILE","syscall":"open","path":"/home/runner/node_modules/polyfill-library/polyfills/Intl/~locale/dua-CM/detect.js"}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! polyfill-library@3.27.4 build: `npm run clean && node tasks/node/updatesources && node tasks/node/buildsources`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the polyfill-library@3.27.4 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2019-02-12T10_14_22_315Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! polyfill-library@3.27.4 postinstall: `npm run build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the polyfill-library@3.27.4 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2019-02-12T10_14_52_881Z-debug.log


Repl.it: package installation failure

Error: Cannot find module 'polyfill-library'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:548:15)
    at Function.Module._load (internal/modules/cjs/loader.js:475:25)
    at Module.require (internal/modules/cjs/loader.js:598:17)
    at require (internal/modules/cjs/helpers.js:11:18)
    at evalmachine.<anonymous>:1:25
    at Script.runInContext (vm.js:74:29)
    at Object.runInContext (vm.js:182:6)
    at evaluate (/run_dir/repl.js:133:14)
    at ReadStream.<anonymous> (/run_dir/repl.js:116:5)
    at ReadStream.emit (events.js:180:13)
```

--- a/tasks/node/updatesources.js
+++ b/tasks/node/updatesources.js
@@ -1,6 +1,6 @@
 'use strict';
 
-const fs = require('fs');
+const fs = require('graceful-fs');
 const path = require('path');
 const denodeify = require('denodeify');
 const glob = denodeify(require('glob'));

